smt/expr.cpp

```diff
@@ -1579,6 +1609,16 @@ expr expr::mkLambda(const set<expr> &vars, const expr &val) {
   return Z3_mk_lambda_const(ctx(), vars.size(), vars_ast.get(), val());
 }
 
+expr expr::app(const vector<expr> &args) const {
```

Comment:
this doesn't seem to be used anywhere?

Reply:
It is used by Strlen::toSMT which does `decl.app(next_recfn_args)`